src/image.rs

```diff
@@ -11,13 +11,43 @@ use anyhow::{bail, Context, Result};
 
 use crate::fsverity::Sha256HashValue;
 
+pub type StatXattrs = BTreeMap<Box<OsStr>, Box<[u8]>>;
+
 #[derive(Debug)]
 pub struct Stat {
     pub st_mode: u32,
     pub st_uid: u32,
     pub st_gid: u32,
     pub st_mtim_sec: i64,
-    pub xattrs: RefCell<BTreeMap<Box<OsStr>, Box<[u8]>>>,
+    pub xattrs: RefCell<StatXattrs>,
+}
+
+impl Default for Stat {
+    fn default() -> Self {
+        Self {
+            st_mode: u32::MAX,
+            st_uid: u32::MAX,
+            st_gid: u32::MAX,
+            st_mtim_sec: -1,
+            xattrs: RefCell::new(BTreeMap::new()),
+        }
+    }
+}
+
+// Might be a bit weird as xattrs cannot be passed as a value here
+// TODO: Turn this into a simple `impl`
+impl TryFrom<&tar::Header> for Stat {
```

Comment:
This seems like a bit of a layering violation.  The knowledge about tar should probably be limited to the code in the `oci/` directory.  It probably makes sense to implement this there, and not as this trait.